src/frontend/src/flows/manage/recoveryMethodsSection.ts

```diff
@@ -101,8 +147,17 @@ export const recoveryKeyItem = ({
   ];
   return html`
     <li class="c-action-list__item" data-device=${alias}>
+      ${tickmark()}
       <div class="c-action-list__label">${alias}</div>
       ${settingsDropdown({ alias, id, settings })}
     </li>
   `;
 };
+
+const tickmark = (): TemplateResult => {
```

Comment:
if we call the template tick-mark should we call the icon the same? or vice-versa?

Reply:
yeah good point!